### PR TITLE
Jumpstart cs deepdive

### PR DESCRIPTION
**Pull request by yilenpan**
Added some AoA problems

`npm start` will now pick a random prompt, move the spec file into the test folder, copy the prompt.js file to solution.js file.

`npm test` will run specs that have been placed in the test folder

`npm run clear` will clear out the test folder

### CODE REVIEW

**Comment by banunatina** [comment]
Sweet! Let me know when this is ready to be merged and I'll also update the README.

Does the new `npm start` handle if there is no spec file?

Also don't forget to add yourself as a collaborator in the package.json.

File: package.json [hunk 1]
```diff
@@ -4,8 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "clear": "rm -f ./test/*.js",
     "test": "mocha test",
-    "start": "vim $(gshuf -n1 -e ./prompts/*/prompt.js)"
+    "start": "prompt=$(echo `gshuf -n1 -e ./prompts/*/` | sed 's/\\.\\/prompts\\///' | sed 's/.\\{1\\}$//');clear;cp prompts/$prompt/prompt.js prompts/$prompt/solution.js;cp specs/$prompt.js test/$prompt.js; vim prompts/$prompt/solution.js"
```
Comment (banunatina) [inline package.json]: I wonder if we can move this out to a separate file eventually and stick `#!/usr/bin/env bash` on top.

**Comment by yilenpan** [comment]
You will need a spec for `npm start` to work - the script works by moving a spec from the specs folder into the test folder (no duplicate prompts).

deprecated `npm run clear` in favor for `npm run reset` which will move all of the specs in test/ back into specs/ and then recursively deletes all of the solutions.js

How do I add myself to the contributors? :0

**Comment by banunatina** [comment]
You can add a contributors key in the package json.

``` json
"contributors": [{ "name" : "Yilen Pan"
                         , "email" : "yilen@yilen.com"
                         , "url" : "http://yilen.com"
                        }],
```

**Comment by yilenpan** [comment]
MERGE MEH

**Pull request merged**